### PR TITLE
Fp 83

### PR DESCRIPTION
**Pull request by Archifer**

### CODE REVIEW

File: android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneWelcomeWrapperErrors.kt [hunk 1]
```diff
@@ -2,24 +2,36 @@ package com.onegini.mobile.sdk.flutter
...
+  NOT_FOUND_SECURITY_CONTROLLER(8004, "The requested Security controller class is not found"), // Android only
+  HTTP_REQUEST_ERROR_INTERNAL(8011, "OneWelcome: HTTP Request failed internally"),
+  HTTP_REQUEST_ERROR_CODE(8011, "OneWelcome: HTTP Request returned an error code. Check Response for more info"),
+  ONEWELCOME_SDK_NOT_INITIALIZED(8012, "OneWelcomeSDK is not initialized"), // Android only
+  CONFIG_ERROR(8032, "Something went wrong while setting the configuration"), // Android only
+  NOT_IN_PROGRESS_REGISTRATION(8034, "Registration is currently not in progress"),
+  NOT_IN_PROGRESS_AUTHENTICATION(8034, "Authentication is currently not in progress"),
+  NOT_IN_PROGRESS_FINGERPRINT_AUTHENTICATION(8034, "Fingerprint Authentication is currently not in progress"),
+  NOT_IN_PROGRESS_OTP_AUTHENTICATION(8034, "OTP Authentication is currently not in progress"),
+  NOT_IN_PROGRESS_PIN_CREATION(8034, "Pin Creation is currently not in progress"), // Android only
```
Comment (MichielVrins) [inline android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneWelcomeWrapperErrors.kt:19]: this one is not android only

File: android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneWelcomeWrapperErrors.kt [hunk 2]
```diff
@@ -2,24 +2,36 @@ package com.onegini.mobile.sdk.flutter
...
-  HTTP_REQUEST_ERROR(8011, "OneWelcome: HTTP Request failed internally"),
-  ERROR_CODE_HTTP_REQUEST(8013, "OneWelcome: HTTP Request returned an error code. Check Response for more info"),
+  NOT_AUTHENTICATED_USER(8002, "There is currently no User Profile authenticated"),
+  NOT_AUTHENTICATED_IMPLICIT(8002, "The requested action requires you to be authenticated implicitly"),
+  NOT_FOUND_USER_PROFILE(8004, "The requested User profile is not found"),
+  NOT_FOUND_AUTHENTICATOR(8004, "The requested authenticator is not found"),
+  NOT_FOUND_IDENTITY_PROVIDER(8004, "The requested identity provider is not found"), // Android only
+  NOT_FOUND_SECURITY_CONTROLLER(8004, "The requested Security controller class is not found"), // Android only
+  HTTP_REQUEST_ERROR_INTERNAL(8011, "OneWelcome: HTTP Request failed internally"),
+  HTTP_REQUEST_ERROR_CODE(8011, "OneWelcome: HTTP Request returned an error code. Check Response for more info"),
```
Comment (MichielVrins) [inline android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneWelcomeWrapperErrors.kt:12]: Message: "OneWelcome: HTTP Request returned an http error code. Check Response for more info"),

File: android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneWelcomeWrapperErrors.kt (hunk 1 above)
Comment (Archifer) [reply android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneWelcomeWrapperErrors.kt]: done

File: android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneWelcomeWrapperErrors.kt (hunk 2 above)
Comment (Archifer) [reply android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneWelcomeWrapperErrors.kt]: done and slightly further updatwed

File: android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneWelcomeWrapperErrors.kt [hunk 3]
```diff
@@ -2,36 +2,31 @@ package com.onegini.mobile.sdk.flutter
...
   ACTION_NOT_ALLOWED_BROWSER_REGISTRATION_CANCEL(
-    8042,
+    8058,
     "Canceling the Browser registration right now is not allowed." +
         " Registration is not in progress or pin creation has already started"
   ),
-  BIOMETRIC_AUTHENTICATION_NOT_AVAILABLE(8043, "Biometric authentication is not supported on this device"),
+  CONFIG_ERROR(8059, "Something went wrong while setting the configuration"), // Android only
+  BIOMETRIC_AUTHENTICATION_NOT_AVAILABLE(8060, "Biometric authentication is not supported on this device"),
   UNEXPECTED_ERROR_TYPE(8999, "An unexpected error type was returned"), // Android only
```
Comment (MichielVrins) [inline android/src/main/kotlin/com/onegini/mobile/sdk/flutter/OneWelcomeWrapperErrors.kt:31]: can you add a comment here that this is only used internally for testing? It confused me for a moment.

**Pull request merged**